hack/vendor.sh

```diff
@@ -24,6 +24,7 @@ clone git github.com/docker/go-units 651fc226e7441360384da338d0fd37f2440ffbe3
 clone git github.com/docker/go-connections v0.1.2
 clone git github.com/docker/engine-api v0.2.1
 clone git github.com/RackSec/srslog 6eb773f331e46fbba8eecb8e794e635e75fc04de
+clone git github.com/imdario/mergo bc0f15622cd2a38ef83388501e4cd6747665b164
```

Comment:
@calavera can you change this to use the tag? https://github.com/imdario/mergo/tree/0.2.1

@imdario tagged that just for us! :dancer: